docs/src/main/asciidoc/_admin-guide/Management_resources.adoc

```diff
@@ -123,17 +147,35 @@ details of one of them, via the CLI one would:
         "request-properties" => {"url" => {
             "type" => STRING,
             "description" => "The URL at which the deployment content is available for upload to the domain's or standalone server's deployment content repository.. Note that the URL must be accessible from the target of the operation (i.e. the Domain Controller or standalone server).",
+            "expressions-allowed" => false,
             "required" => true,
-            "min-length" => 1,
-            "nillable" => false
+            "nillable" => true,
+            "alternatives" => [
+                "input-stream-index",
+                "hash",
+                "bytes",
+                "path",
+                "relative-to",
+                "empty"
+            ],
+            "min-length" => 1L,
+            "max-length" => 2147483647L,
+            "web-url" => true,
+            "stability" => "default"
         }},
         "reply-properties" => {
             "type" => BYTES,
             "description" => "The hash of managed deployment content that has been uploaded to the domain's or standalone server's deployment content repository.",
-            "min-length" => 20,
-            "max-length" => 20,
-            "nillable" => false
```

Comment:
Is this intentional deletion? 

`"nillable" => false,` is included in the CLI output.

Reply:
@soul2zimate You're right.

Reply:
Fixed.

Reply:
Thanks for noticing this; that's what got me thinking about how the request-properties weren't correct.